### ISSUE TITLE
shouldn't discriminator label be all ones for real data and zeros for fake data?

### ISSUE THREAD

**Issue by norris9410** [deepak112/Keras-SRGAN#28]
Hi,

First thank you very much for your work and it's very easy to read and understand. 
When I am going through the code, seems like the discriminator labels are bit confusing to me. 

Specifically, in train.py line 70&71, 
`real_data_Y = np.ones(batch_size) - np.random.random_sample(batch_size)*0.2`
`fake_data_Y = np.random.random_sample(batch_size)*0.2`

Should it be like:
real_data_Y = np.ones(batch_size)
fake_data_Y = np.zeros(batch_size)

Hope to get the response. Thanks in advance!

**Comment by alfinnurhalim** [comment]
I think he made relative Y. instead of binary "1" for true and "0" for fake, he convert it to float. the closer it to 1, the more it likely to be "Real" data. this is pretty common in binary classification task, this proved encourage the network to learn harder.

**Comment by deepak112** [comment]
Yes you can use 1 and 0 as labels. By converting it to float it makes network to learn harder.
